src/core_atmosphere/physics/physics_wrf/module_bl_ysu.F

```diff
@@ -537,6 +537,8 @@ subroutine ysu2d(j,ux,vx,tx,qx,p2d,p2di,pi2d,                               &
                                                                      buoy_ysu
    real,    dimension( ims:ime )             ::                       pblh_ysu,&
                                                                       vconvfx
+   real,    dimension( kts:kte )   :: dummy1,dummy2,dummy4
+   real,    dimension( kms:kme )   :: dummy3
```

Comment:
Since these `dummyN` variables each have a specific use, we could consider giving them names like `thvx_1d`, `tke_ysu_1d`, `zq_1d`, and `dzq_1d`.

Reply:
I totally agree that it will add more clarity to our modifications to the original sourcecode.